.github/workflows/pre-release-check.yml

```diff
@@ -0,0 +1,40 @@
+name: pre-release-check
+
+env:
+  ENCRYPTION_KEY: ${{ secrets.ENCRYPTION_KEY }}
+  GOOGLE_APPLICATION_CREDENTIALS: scripts/data-integration-test-key.json
+  GOOGLE_PROJECT_ID: data-integration-test
+  SBT_OPTS: -DbeamRunners=DataflowRunner
+  DEFAULT_DATAFLOW_ARGS: --runner=DataflowRunner --project=data-integration-test --region=europe-west1
+
+on:
+  workflow_dispatch # Manually triggered
+jobs:
+  test-dataflow:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: cache SBT
+        uses: coursier/cache-action@v6
+      - name: Java setup
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: ${{matrix.java}}
+      - name: Setup project
+        run: scripts/gha_setup.sh
+      - name: (Dataflow) IO Writes
```

Comment:
wdyt of keep this simple by not making a distinction? Job name (`--jobName`) could be sufficient here. My main thought is jobs that might do writing & reading

Reply:
that's a good point! I did it this way because, in this specific AvroExample,  the `--specificOut` configuration needs to run first, since its output will be fed into the `--specificIn` job's input. So there's some notion of dependencies between jobs 🤔 

Maybe we could write specific jobs for these prelease tests that are designed to have self-contained dependencies, like:

```scala
object DataflowPreReleaseAvroTest {
  def main(args: Array[String])
     val (sc, _) = ...
     // do an Avro write
     sc.run.waitUntilDone
 
     val (sc2, _) = ...
     // do an Avro read using the data that was written
     sc2.run.waitUntilDone 
}